src/classes/OPP_OpportunityNaming_BATCH.cls

```diff
@@ -34,22 +34,39 @@
 * @group-content ../../ApexDocContent/Opportunity.htm
 * @description Batch class creates names all Opportunities per the naming spec.
  */
-public class OPP_OpportunityNaming_BATCH implements Database.Batchable<sObject>, Schedulable {
```

Comment:
I think it would be helpful to update the block comment on the class to explain that the batch job now uses a chunking pattern.